README.md

```diff
@@ -6,7 +6,47 @@ hide:
 
 # CivicActions Guidebook
 
-[![Documentation Status](https://readthedocs.org/projects/civicactions-handbook/badge/?version=latest)](https://guidebook.civicactions.com/en/latest/?badge=latest)
+<!--[![Documentation Status](https://readthedocs.org/projects/civicactions-handbook/badge/?version=latest)](https://guidebook.civicactions.com/en/latest/?badge=latest) -->
```

Comment:
@garaynakca if @grugnog agrees we can remove this or move it to a different page.

Reply:
I think we could just remove this, honestly - showing green badges/coverage etc is nice for an open source project that wants to encourage usage, but I am not sure it is that relevant to a documentation project like this.

Reply:
<img width="480" alt="Screen Shot 2023-02-24 at 11 12 13 AM" src="https://user-images.githubusercontent.com/102678497/221270690-64754948-7539-438e-884f-e3caabd8878b.png">
@dmundra I think @ContentBrewmaster wants to remove this "beta" in the header.  Can you point me in the right direction to do that.  Thanks!

Reply:
Ya @garaynakca. That comes from https://github.com/CivicActions/guidebook/blob/master/.config/mkdocs.yml#L1

Reply:
@garaynakca any blockers for merging the new design that I can help with?

Reply:
I went ahead and removed 'Documentation Status'.